### PR TITLE
08 patch api articles article

### PR DESCRIPTION
**Pull request by SoranRaof**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -292,14 +314,44 @@ test('ignore extra properties with a status 201', () => {
...
+    })
+    test('responds with a 404 along without a no article found message when given an invalid article id', () => {
+        return request(app)
+        .get('/api/articles/999999')
+        .expect(404)
+        .then(res => {
+            const { msg } = res.body;
+            expect(msg).toBe('Article not found');
+        })
+    })
```
Comment (sirKevlar) [inline __tests__/api.test.js:356]: Just one sad path test missing: inc_votes could be the wrong data type {inc_votes: 'not-a-number'}

File: __tests__/api.test.js [hunk 2]
```diff
@@ -151,6 +151,28 @@ describe('POST /api/articles/:article_id/comments', () => {
     })
 })
 
+describe('PATCH /api/articles/:article_id', () => {
+    test('responds with a 200 status code and updated article', () => {
+        const votes = {
+            inc_votes: 1
```
Comment (HannahBurks) [inline __tests__/api.test.js:157]: Here we need to add three more tests and make sure we are error handling for the following:
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999
Status 400, missing / incorrect body, e.g. inc_votes property is not a number, or missing

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Hey Soran, great start - just some extra tests and error handling needs to be added here :) Once added please send another PR

File: __tests__/api.test.js (hunk 2 above)
Comment (SoranRaof) [reply __tests__/api.test.js:157]: Hello Hannah, I believe I've done these tests at the bottom of my test suite?

**Pull request closed without merging**